### PR TITLE
Update Kusto schema

### PR DESCRIPTION
**Pull request by radennis**

### CODE REVIEW

**Comment by radennis** [comment]
@Tiano2017 could you please take a look at this PR?

File: schemas/2019-01-21/Microsoft.Kusto.json [hunk 1]
```diff
@@ -230,13 +244,16 @@
       "description": "Class representing the Kusto cluster properties."
     },
     "DatabaseProperties": {
+      "type": "object",
       "properties": {
         "softDeletePeriod": {
           "type": "string",
+          "format": "duration",
```
Comment (TianoMS) [inline schemas/2019-01-21/Microsoft.Kusto.json]: >"format": "duration", [](start = 10, length = 21)

Visual Studio doesn't support the "duration" format. can we just  remove this lines?

Comment (radennis) [reply schemas/2019-01-21/Microsoft.Kusto.json]: Removed

**Comment by radennis** [comment]
@Tiano2017 could you please approve?

**Comment by radennis** [comment]
@Tiano2017 it's been a while, would you please approve?

**Pull request merged**